wikiteam3/dumpgenerator/generator.py

```diff
@@ -226,6 +227,9 @@ def resumePreviousDump(config={}, other={}):
                 f.close()
             except FileNotFoundError:
                 pass  # probably file does not exists
+            if len(images)>0 and len(images[0]) < 5:
+                print("Warning: Detected old image list format. You can delete 'images.txt' manually and restart the script.")
```

Comment:
Alternatively, you can manually delete the old images.txt and then regenerate the images list. This operation is safe and will not re-download the image/file if it passes the sha1sum.

Reply:
Please add some instructions to Readme.md for new/changed features.